README.md

```diff
@@ -131,31 +131,38 @@ Depending on your preference, you can also add parameters as named arguments, in
 response = await dg_client.transcription.prerecorded(source, punctuate=True, keywords=['first:5', 'second'])
 ```
 
-## Code Samples
-
-To run the sample code, you may want to create a virtual environment to isolate your Python projects, but it's not required. You can learn how to make and activate these virtual environments in [this article](https://blog.deepgram.com/python-virtual-environments/) on our Deepgram blog.
-
-#### Streaming Audio Code Samples
+## Testing
 
-In the `sample-projects` folder, there are examples from four different Python web frameworks of how to do live streaming audio transcription with Deepgram. These include:
+### Setup
 
-- Flask 2.0
-- FastAPI
-- Django
-- Quart
+```
+pip install pytest
+pip install pytest-cov
+```
 
-## Testing
+### Run All Tests
 
 ```
 pytest --api-key <key> tests/
```

Comment:
I would love it if I didn't have to pass the API key in on the command line to run the tests. If the argument is missing, can we default to having the code check the environment variable `DEEPGRAM_API_KEY`?

This change doesn't need to be done in this PR (or at all if other people feel differently), so feel free to resolve this comment with no changes if you want.

Reply:
I'll have to look into making it work that way. No promises. 🙂